h2o-core/src/test/java/water/runner/LocalTestRuntime.java

```diff
@@ -0,0 +1,16 @@
+package water.runner;
+
+import org.junit.Ignore;
+import water.Key;
+
+import java.util.HashSet;
+import java.util.Set;
+
+@Ignore
+class LocalTestRuntime {
+    protected static Set<Key> initKeys;
+
+    static{
```

Comment:
unnecessary static block

Reply:
Adjusted to your code style preferences.